mobileapp/package.json

```diff
@@ -17,6 +17,7 @@
   },
   "dependencies": {
     "@capacitor/android": "^4.6.2",
+    "@capacitor/app": "^4.1.1",
```

Comment:
Should this not use the same minor version as the other `@capacitor/*` packages?

Reply:
No I think they're each their own independent packages